corehq/apps/receiverwrapper/rate_limiter.py

```diff
@@ -67,6 +67,45 @@ def _get_per_user_submission_rate_definition(domain):
 )
 
 
+global_case_rate_limiter = RateLimiter(
+    feature_key='global_case_updates',
+    get_rate_limits=lambda: get_dynamic_rate_definition(
+        'global_case_updates',
+        default=RateDefinition(
+            per_hour=170000,
```

Comment:
if these are meant to be relative to the form limits could we make those a constant: 

`FORMS_PER_HOUR_GLOBAL * 10`